waiter/src/waiter/process_request.clj

```diff
@@ -385,7 +387,13 @@
                               (statsd/inc! metric-group "response_bytes" unreported-bytes)
                               bytes-streamed')
                             bytes-reported-to-statsd))]
-                    (when more-bytes-possibly-available?
+                    (if-not more-bytes-possibly-available?
+                      (let [request (-> request
+                                        (utils/mark-request-time :closed))]
+                        (rlog/log (assoc (rlog/request->context request)
+                                         :bytes-streamed bytes-streamed
+                                         :status (:status response)
+                                         :termination-state :success)))
```

Comment:
Why not have `rlog/log` take in the `request` directly and call `rlog/request->context` instead of making the caller do it?

Reply:
Done